### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by diversantvlz**
Picture cached files and javascript dump files i removed from the repository. (They are generated by the "wellcommerce:install" command, I do not see the need for them. They interfere with the merge process).

Added "news_feed" layout box in CmsBundle.

Fix template and translate.

Reordering register bundles for use custom users bundles from src. I saw WellCommerce\Bundle\CoreBundle\Locator::getBundleClasses() line 31 $localBundles = [];
I understand this is a stub to register bundles from src?

p.s: Sorry for my english

### CODE REVIEW

**Comment by adampiotrowski** [comment]
@diversantvlz It looks like I'll have a busy weekend making a CR 😍 😍 

For bundle registration system I have a different solution because we need that the bundles are added/removed during the package's installation/removal so it requires many changes in ScriptHandler (script used as composer hook) and running additional checks/migrations during the installation/update . I'll check whether it can be merged or reused.

Thanks a lot!

**Comment by diversantvlz** [comment]
I made a wrapper for your Bundle registration system, so that AppKernel could be customized (Symfony styleguid),
look WellCommerce/Bundle/StandardEditionBundle/HttpKernel/AbstractWellcommerceKernel and AppKernel.
Your decision has remained working and it can be further developed as you please.

File: src/WellCommerce/Bundle/CmsBundle/DataSet/Front/NewsDataSet.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ public function configureOptions(DataSetConfiguratorInterface $configurator)
         
         $configurator->setColumnTransformers([
             'route' => $this->manager->createTransformer('route'),
-            'photo' => $this->manager->createTransformer('image_path', ['filter' => 'original']),
+            /*'photo' => $this->manager->createTransformer('image_path', ['filter' => 'original']),*/
```
Comment (adampiotrowski) [inline src/WellCommerce/Bundle/CmsBundle/DataSet/Front/NewsDataSet.php]: @diversantvlz What's the reason of blocking it?

File: src/WellCommerce/Bundle/StandardEditionBundle/DataFixtures/ORM/LoadNewsData.php [hunk 2]
```diff
@@ -39,6 +40,11 @@ public function load(ObjectManager $manager)
                 'type' => 'News',
                 'name' => 'News',
             ],
+            'news_feed' => [
+                'type' => 'NewsFeed',
+                'name' => 'News Freed',
```
Comment (adampiotrowski) [inline src/WellCommerce/Bundle/StandardEditionBundle/DataFixtures/ORM/LoadNewsData.php]: @diversantvlz Please change to News Feed

File: src/WellCommerce/Bundle/CmsBundle/DataSet/Front/NewsDataSet.php (hunk 1 above)
Comment (adampiotrowski) [inline src/WellCommerce/Bundle/CmsBundle/DataSet/Front/NewsDataSet.php]: Ah, I see. You changed the template, right? 

{{ image(news.photo.path, 'medium') }}

**Pull request merged**